pom.xml

```diff
@@ -182,12 +181,47 @@
 					<artifactId>spotbugs-maven-plugin</artifactId>
 					<version>4.6.0.0</version>
 				</plugin>
+				<plugin>
+					<groupId>org.jacoco</groupId>
+					<artifactId>jacoco-maven-plugin</artifactId>
+					<version>0.8.8</version>
+				</plugin>
```

Comment:
Jacoco plugin is instrumenting the code when we run tests so that it can detect in which line of code the executions (tests) went through. This can be handy when we have a bug, we can create a test to validate the bug exists and then see what we missed in our code.